ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -542,6 +538,10 @@ public DdlCommandExec getDdlCommandExec() {
     return ddlCommandExec;
   }
 
+  public KsqlConfig getInitializationKsqlConfig() {
```

Comment:
Similar to my other comment. This should just be called `getServiceId` and return a `String`. Would be better from an encapsulation point of view.

Reply:
Good point. I removed `initializationKsqlConfig` as a member variable and use `serviceId` instead. This is now `getServiceId()`.